src/wp-includes/general-template.php

```diff
@@ -5050,6 +5059,15 @@ function wp_required_field_message() {
 		sprintf( __( 'Required fields are marked %s' ), wp_required_field_indicator() )
 	);
 
+	/**
+	 * Filters the message to explain required form fields.
+	 *
+	 * @since 6.1.0
+	 *
+	 * @param string $message Message text and glyph wrapped in a `span` tag.
+	 */
+	$message = apply_filters( 'wp_required_field_message', $message );
+
 	return $message;
```

Comment:
```suggestion
	return apply_filters( 'wp_required_field_message', $message );
```
This filter seems like it should be the final value. What do you think @kebbet and @audrasjb?

Reply:
I have nothing against those changes, can update the PR. Fewer lines of code is indeed cleener.